wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java

```diff
@@ -312,8 +333,8 @@ public String calculateSha1ByNameVersionArchitectureAndType(String name, String
     /**
      * Calculates SHA-1 for library by name, version and dependencyType
      *
-     * @param name of library
-     * @param version of library
+     * @param name           of library
```

Comment:
why do we need these spaces?

Reply:
This what happens when I press ctrl+shift+L -> Its to fix file format 
Do you want to leave it like this or revert it?

Reply:
revert please